simple/run.sh

```
@@ -1,21 +1,48 @@
+
+sleep 8
+# koa2
+curl 'http://127.0.0.1:7001/'
+# egg3
+curl 'http://127.0.0.1:7030/'
+curl 'http://127.0.0.1:7031/'
+curl 'http://127.0.0.1:7032/'
+# egg4
+curl 'http://127.0.0.1:7040/'
 
 test `tail -c 1 $CSV` && printf "\n" >> $CSV
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)